### PR TITLE
[BX-1101] NFT Details - Core

### PR DESCRIPTION
**Pull request by derHowie**
Figma link (if any): https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/Rainbow-BX?type=design&node-id=8127-263102&mode=design&t=4TzQLRg3kLaulHxR-0

## What changed (plus any additional context for devs)
This PR adds the base UI for NFT details.

Notable exceptions:
- no additional UI for ENS details
- share button is still TBD w/ design
- owners/%owners will be a standalone ticket
- hide/report will be a standalone ticket
- all time high / all time low has been scratched

## Screen recordings / screenshots
PoW dark mode: https://recordit.co/3VhaA2C3RX
PoW light mode: https://recordit.co/IHYim7s60H

## What to test
Ensure details screen reflects designs and functions correctly

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
💯  x 💯

File: src/entries/popup/pages/home/NFTs/NFTDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,1039 @@
...
+  );
+};
+
+const NFTAccordionAboutSection = ({ nft }: { nft?: UniqueAsset | null }) => {
+  const network = useMemo(() => {
+    if (nft?.network === 'mainnet') {
+      return 'Ethereum';
+    }
+    return nft?.network && capitalize(nft.network);
+  }, [nft?.network]);
```
Comment (greg-schrammel) [inline src/entries/popup/pages/home/NFTs/NFTDetails.tsx:388]: I think it's more expensive to `useMemo` this then just
```suggestion
  const network = nft?.network === 'mainnet' ? 'Ethereum' : capitalize(nft.network);
```

**Review by greg-schrammel — APPROVED** [review]
prettyyy

File: src/entries/popup/pages/home/NFTs/NFTDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,1039 @@
...
+        url: `https://etherscan.io/address/${deployedBy}`,
+      }),
+    [],
+  );
+  const copyTokenContract = useCallback((contractAddress: Address) => {
+    navigator.clipboard.writeText(contractAddress as string);
+    triggerToast({
+      title: i18n.t('nfts.details.address_copied'),
+      description: truncateAddress(contractAddress),
+    });
```
[the remark this answers is not in the document]
Comment (derHowie) [reply src/entries/popup/pages/home/NFTs/NFTDetails.tsx]: good call, also made this change

**Pull request merged**